src/renderer/features/assets/AssetsPortfolioView/ui/TokenBalance.tsx

```diff
@@ -0,0 +1,59 @@
+import { useUnit } from 'effector-react';
+
+import { useI18n } from '@app/providers';
+import { Icon, Tooltip, BodyText, Plate, FootnoteText, CaptionText } from '@shared/ui';
+import type { AssetByChains } from '@shared/core';
+import { TokenPrice } from '@entities/price';
+import { AssetIcon, AssetLinks } from '@entities/asset';
+import { networkModel } from '@entities/network';
+import { ChainIcon } from '@entities/chain';
+import { AssembledAssetAmount } from './AssembledAssetAmount';
+
+type Props = {
+  asset: AssetByChains;
+};
+
+export const TokenBalance = ({ asset }: Props) => {
+  const { t } = useI18n();
+  const chain = asset.chains[0];
+
+  const chains = useUnit(networkModel.$chains);
+
+  return (
+    <Plate
+      className="p-0 z-10 h-[52px] w-full items-center flex pl-[30px] pr-2 
+    transition-shadow hover:shadow-card-shadow focus:shadow-card-shadow"
+    >
+      <div className="flex gap-x-2 flex-1">
+        <div className="flex items-center gap-x-2">
+          <AssetIcon src={asset.icon} name={asset.name} />
+          <div>
+            <BodyText>{chain.assetSymbol}</BodyText>
+            <div className="flex items-center gap-x-1.5 mr-3">
+              <FootnoteText className="text-text-tertiary">{chain.name}</FootnoteText>
+              <ChainIcon src={chains[chain.chainId].icon} name={chain.name} size={18} />
+              {chain.balance?.verified && (
+                <div className="flex items-center gap-x-2 text-text-warning">
+                  {/* FIXME: tooltip not visible when first displayed network invalid. For now just render it below icon */}
```

Comment:
Can we create an issue and eventually mention it here? Such FIXME, unless they are tackled right away, tend to stay in the code forever.